server/middleware/middleware.go

```diff
@@ -295,6 +296,10 @@ func (i *NobodyIdentityService) GetServiceTokenID() string {
 	return ""
 }
 
+func (i *NobodyIdentityService) IsTenantIDValid(id string) error {
+	return fmt.Errorf("%s tenant_id is invalid", id)
```

Comment:
I'm not sure if this is correct - `NobodyIdentityService` returns "Nobody" as tenant id
https://github.com/cloudwan/gohan/pull/745/files#diff-f2c62ab5c95abc1469d5f5a41bf5321cL262

Reply:
At least, error message should be changed. If we do not want to expect `IsTenantIDValid` operation for nobody auth case, something like `invalid operation` error message should be returned.